### DISCUSSION TITLE
Remove Linux Support Description from Plover Website

### DISCUSSION THREAD

**Discussion by ghost** [opensteno/plover#1627] [Ideas]
#### Issue Description
Currently, the Plover website includes a description of Linux support, stating that Plover can be used on Linux systems with Xorg as the video server. However, with the shift towards Wayland as the main Linux video server and the reduced development activity of Xorg(Xorg currently is in maintenance mode), it is necessary to update the website to reflect the current state of Linux support in Plover.

#### The Problem:

1. Outdated Information: The current website description of Linux support in Plover is misleading and no longer accurate due to the shift towards Wayland as the primary video server on Linux systems.
2. Incompatibility with Wayland: Plover does not currently support Wayland, which is the primary video server for many modern Linux distributions. Therefore, the presence of Linux support on the website might mislead users into thinking that Plover is compatible with Wayland when it is not.

#### Proposed Solution

1. Remove Linux Support Description: The website should be updated to remove any references to Linux support, specifically mentioning Xorg as the video server.
1. Clarify System Requirements: The updated website should clearly state the system requirements for Plover, explicitly mentioning that Plover does not support Wayland at this time.

#### Benefits:

1. Accurate Information: By removing the outdated Linux support description, potential users will have up-to-date and accurate information about the current state of Linux support in Plover.
2. Avoiding Misleading Users: Removing the Linux support description will prevent users from assuming that Plover is compatible with Wayland, saving them from potential frustration and confusion.
3. Focus on Supported Platforms: By removing Linux support from the website, the development team can focus on maintaining and improving support for the officially supported platforms, ensuring a better user experience.

#### Additional Considerations:

1. Future Wayland Support: If there are plans to support Wayland in the future, it would be helpful to mention this on the website, along with an estimated timeline.
2. Documentation Update: The corresponding sections in the Plover documentation should be updated to reflect the removal of Linux support from the website.

**Comment by Willem3141** [comment]
Plover supports Linux, so obviously Plover's website states that it supports Linux. I don't see a problem here. Removing the mention of Linux support would be misleading and unhelpful for those using X11.

I do think it may be good to add a more prominent note that Plover doesn't work on Wayland.

**Reply by ghost** [reply]
Support for an outdated display server that is no longer actively developed, but only in maintenance mode, is not platform support.  By your logic, Adobe Acrobat supports Linux.

**Reply by Willem3141** [reply]
Let's not be absurd; Adobe Acrobat for Linux hasn't received any updates, including security updates, since 2013. Are you suggesting that Plover offers a similar level of support to that?

More seriously: yes, I would love Wayland support for Plover (it's [being worked on](https://github.com/openstenoproject/plover/pull/1461), by the way!) as it would bring me one step closer to actually being able to switch to Wayland for my main system. But no, Wayland is not the only valid choice on Linux; let's not pretend it is.

**Reply by ghost** [reply]
> Let's not be absurd; Adobe Acrobat for Linux hasn't received any updates, including security updates, since 2013. Are you suggesting that Plover offers a similar level of support to that?
> 
> More seriously: yes, I would love Wayland support for Plover (it's [being worked on](https://github.com/openstenoproject/plover/pull/1461), by the way!) as it would bring me one step closer to actually being able to switch to Wayland for my main system. But no, Wayland is not the only valid choice on Linux; let's not pretend it is.

jtroo/kanata implemented kernel-level chords on Windows and Linux (including Wayland and X).

keyd implemented kernel-level chords on Linux and user level on Xorg, and wlroots.

What's wrong with plover?

**Reply by panathea** [reply]
No offence meant to you personally, but this is a small open source project, and for our maintainers' sanity expect contributions to be helpful and positive. Your tone, whether you intend it or not, comes off as rude and gatekeepey. I'm not an active maintainer, but if I were, I wouldn't appreciate the ways you've framed your feedback. Have a great day.

**Reply by ghost** [reply]
> No offence meant to you personally, but this is a small open source project, and for our maintainers' sanity expect contributions to be helpful and positive. Your tone, whether you intend it or not, comes off as rude and gatekeepey. I'm not an active maintainer, but if I were, I wouldn't appreciate the ways you've framed your feedback. Have a great day.

Keyd and Kanata are even smaller.
